### PR TITLE
Bug 1903192: - Role name missing on create role binding form

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @spadgett 

Screen Shot 2020-12-04 at 3 03 24 PM
Screen Shot 2020-12-04 at 3 03 44 PM

### CODE REVIEW

File: frontend/public/components/RBAC/role.jsx [hunk 1]
```diff
@@ -33,9 +33,12 @@ const menuActions = [
   (kind, role) => ({
     label: 'Add Role Binding',
     href: `/k8s/${
-      role.metadata.namespace ? `ns/${role.metadata.namespace}` : 'cluster'
-    }/rolebindings/~new?rolekind=${roleKind(role)}&rolename=${role.metadata.name}${role.metadata
-      .namespace && `&namespace=${role.metadata.namespace}`}`,
+      role.metadata.namespace
+        ? `ns/${role.metadata.namespace}/rolebindings/~new?rolekind=${roleKind(role)}&rolename=${
+            role.metadata.name
+          }${role.metadata.namespace && `&namespace=${role.metadata.namespace}`}`
```
Comment (spadgett) [inline frontend/public/components/RBAC/role.jsx:39]: We already check for `role.metadata.namespace` just above, so we shouldn't need to check again.

Comment (cyril-ui-developer) [reply frontend/public/components/RBAC/role.jsx]: @spadgett Updated, thanks.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**